druid/src/widget/widget_ext.rs

```diff
@@ -115,6 +115,13 @@ pub trait WidgetExt<T: Data>: Widget<T> + Sized + 'static {
         Container::new(self).background(brush)
     }
 
+    /// Wrap this widget in a [`Container`] with the provided foreground `brush`.
+    ///
+    /// See [`Container::foreground`] for more information.
+    fn foreground(self, brush: impl Into<BackgroundBrush<T>>) -> Container<T> {
+        Container::new(self).foreground(brush)
+    }
```

Comment:
The comment on this `foreground` method is now fine. Please do me a favor and also update the comment on the old `background` method that is just before `foreground`. Specifically it still has the incorect text ```with the provided `background`.``` -- please replace it with ```with the provided background `brush`.``` for consistency.